staging/src/k8s.io/api/discovery/v1/generated.proto

```diff
@@ -34,8 +34,9 @@ message Endpoint {
   // addresses of this endpoint. The contents of this field are interpreted
   // according to the corresponding EndpointSlice addressType field. Consumers
   // must handle different types of addresses in the context of their own
-  // capabilities. This must contain at least one address but no more than
-  // 100.
+  // capabilities. This must contain at least one address but no more than 100.
+  // these are all assumed to be fungible and clients may choose to only use the first element.
```

Comment:
Nit: Wrap lines at 80 chars here and elsewhere.

Reply:
thanks, Done